doc-deploy-index/action.yml

```diff
@@ -75,10 +75,10 @@ inputs:
     default: false
     type: string
 
-  pymeilisearchopts:
+  stop-urls:
     description: >
-      Set of options to pass to `pymeilisearch <https://pymeilisearch.docs.ansys.com/version/stable/>`_
-      while scraping. Default is ``None``.
+      A list of stop points when scraping URLs. If specified,
+      crawling will stop when encountering any URL containing any of the strings in this list.
     default: ''
```

Comment:
Wouldn't it be better to just allow all the peimiliesarch possible options like it was done previously?

Reply:
You are right, Roberto. I did not knew this input was available. It looks like it is not being used in the final command.

Let us go with this input and enable it in the command, @Revathyvenugopal162.

Reply:
Sure revert as options and added it in command